app/services/container_dashboard_service.rb

```diff
@@ -123,6 +123,7 @@ def heatmaps
     # Get latest hourly rollup for each node.
     node_ids = @ems.container_nodes if @ems.present?
     metrics = MetricRollup.latest_rollups(ContainerNode.name, node_ids)
+    metrics = metrics.where('timestamp > ?', 1.day.ago.utc).includes(:resource)
```

Comment:
Think `includes(:resource)` is already in MetricRollup. With that select over there, not sure that you'll get what you need.